### PR TITLE
add a test for `orgs` in `/site/user/me`

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner @ryanpbrewster

### CODE REVIEW

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/website/UserControllerTest.scala [hunk 1]
```diff
@@ -69,7 +71,7 @@ class UserControllerTest extends Specification with ShoeboxTestInjector {
               "numConnections":0,
               "numFollowers":0,
               "pendingFriendRequests":0,
-              "orgs": []
+              "orgs": [{"id":"${Organization.publicId(org.id.get)(inject[PublicIdConfiguration]).id}","ownerId":"${user.externalId}","handle":"${org.getHandle.value}","name":"${org.name}","numMembers":${1},"numLibraries":${0}}]
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/UserControllerTest.scala]: What's the `${1}` for?

Comment (camhashemi) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/UserControllerTest.scala]: To tell future-someone that when they add members to the org, this needs to be a variable. We're not testing the values of `numMembers` and `numLibraries` here, but I could do that.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/UserControllerTest.scala]: I see that we just hard-code non-tested values in other places, I could also do that.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/UserControllerTest.scala]: Yeah. you're already hard coding values, making Scala call `1.toString` and concat it back into the string may confuse someone.

**Pull request merged**